grammars/keywords-ar.lark

```diff
@@ -1,46 +1,46 @@
-_PRINT: "قول" | "print"
-_ASK: "اسأل" | "ask"
-_ECHO: "ردد" | "echo"
-_FORWARD: "تقدم" | "forward"
-left: "left"
-right: "right"
-_IS: "هو" | "is"
-_AT: "بشكل" | "at"
-random: "عشوائي" | "random"
-_IN: "في" | "in"
-_IF: "اذا" | "if"
+_PRINT: ("قول" | "print") _SPACE?
```

Comment:
I will have to update my generation script to make it so that this gets added automatically (note to self)